tools/vtx_gfx_extractor.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+import sys
+import struct
+import tempfile
+import subprocess
+import ctypes
+
+# libmio0.so has to compiled be like so:
+# gcc -o libmio0.so libmio0.c -shared -fPIC
+_mio0_decode = ctypes.CDLL("tools/libmio0.so")
```

Comment:
I guess `_mio0_decode` would be better called `_libmio0`, but that's a small detail